isis/tests/BundleMeasureTests.cpp

```diff
@@ -0,0 +1,146 @@
+#include <memory>
+
+#include "BundleControlPoint.h"
+#include "BundleMeasure.h"
+#include "BundleSettings.h"
+#include "ControlMeasure.h"
+#include "ControlPoint.h"
+#include "IException.h"
+#include "SpecialPixel.h"
+
+#include "gmock/gmock.h"
+
+using namespace std;
+
+// To be used for basic setup of a BundleMeasure test object. (Fixture Function)
+class BundleMeasure_CreateObj : public ::testing::Test {
+
+  protected:
+
+    // Declare unique pointers to expose them when using the fixture.
+    unique_ptr<ControlPoint> controlPoint;
+    unique_ptr<ControlMeasure> controlMeasure;
+    unique_ptr<BundleMeasure> testBundleMeasurePtr;
+    unique_ptr<BundleControlPoint> testBundleControlPointPtr;
+
+
+    void SetUp() override {
+
+      // Create a ControlPoint object using the default constructor
+      // to feed its pointer into the BundleControlPoint constructor.
+      controlPoint = unique_ptr<ControlPoint>( new ControlPoint() );
+
+      // Create a ControlMeasure object using the default constructor
+      // to feed its pointer into the BundleMeasure constructor.
+      controlMeasure = unique_ptr<ControlMeasure>( new ControlMeasure() );
+
+      // Create a BundleSettingsQsp to feed it to
+      // the BundleMeasure constructor.
+      BundleSettingsQsp bundleSettingsQsp( new BundleSettings );
+
+      // The pointers are then linked to newly-created BundleMeasure and
+      // BundleControlPoint objects.
+      testBundleControlPointPtr = unique_ptr<BundleControlPoint>( new BundleControlPoint( bundleSettingsQsp,
+                                                                                     controlPoint.get() ) );
+
+      testBundleMeasurePtr = unique_ptr<BundleMeasure>( new BundleMeasure( controlMeasure.get(),
+                                                                           testBundleControlPointPtr.get() ) );
+
+    }
+};
+
+TEST_F( BundleMeasure_CreateObj, Constructor ) {
+  EXPECT_FALSE( testBundleMeasurePtr->isRejected() );
+
+  EXPECT_EQ( NULL, testBundleMeasurePtr->camera() );
```

Comment:
You should use `nullptr` instead of `NULL`

Reply:
When I was talking to Adam and Stuart about the best way to test these methods, they felt that testing any methods other than the one I have PR'd would be essentially the same as testing ControlMeasure and BundleControlPoint over again. 

I can certainly still implement tests for these methods, but I'm not exactly sure which camera and image I should be using for testing purposes.

I will be sure to change the NULL to nullptr. Would it only be necessary to change it for the camera method, or are there other places that should also use it?

Reply:
Looking at the code that seems reasonable